e2e/use-router.spec.ts

```diff
@@ -126,9 +131,15 @@ test.describe(`useRouter`, async () => {
           msgs.push(text.slice(prefix.length));
         }
       });
+      await waitForHydration(page);
       await page.click('text=Static router.push button');
       await expect(page.getByRole('heading', { name: 'Static' })).toBeVisible();
-      expect(msgs).toEqual(['Route change started', 'Route change completed']);
+      expect(msgs).toEqual(
+        expect.arrayContaining([
```

Comment:
Does this allow the reverse order like `['Route change completed', 'Route change started']`?

Reply:
This can be reverted too. I think it was previously different logs were mixed.